etrago/cluster/electrical.py

```diff
@@ -166,7 +166,7 @@ def find_de_closest(network, bus_ne):
         "H2_saltcavern": "power_to_H2",
         "dsm": "dsm",
         "Li ion": "BEV charger",
-        "Li_ion": "BEV_charger",
```

Comment:
Li_ion is the carrier that is used in eGon2035 (and we also just changed it in eGon100RE), so it should not be deleted here

Reply:
acf4ca46f58164ed5d17989ce1177a30c5373fde